### PR TITLE
Bump to v0.2.0

### PR DESCRIPTION
**Pull request by hakamine**
Bump version so that pip upgrades to latest version of the code

### CODE REVIEW

**Comment by sevein** [comment]
Does this fix the problem where the package was not being updated when installing directly from git? I thought it was related to the fact that the version of pip was very old.

**Comment by hakamine** [comment]
One of the purposes of changing the version number is to have an easy way to know what has been installed on a system (version has been 0.1 for a while even though there were code updates, so two systems may show the same version 0.1 installed while functionality is different). I would like to suggest to increase the version number each time something new is merged to the master branch.
The other purpose was to force pip to upgrade, as I thought that pip didn't do anything when it found the same version number already installed, however when I tested doing a `pip install --upgrade git+git` (with the pip 1.5.4 version provided by trusty) it uninstalled the existing module and installed from the repo even when the version number in the repo was less than what was already installed (not sure if this is because of using `pip git+git` or of the pip version being old )

File: setup.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 setup(
     name="agentarchives",
     description="Clients to retrieve, add, and modify records from archival management systems",
-    author="Misty De Meo",
-    author_email="mdemeo@artefactual.com",
+    author="Artefactual Systems",
+    author_email="info@artefactual.com",
     license="AGPL 3",
-    version="0.1",
+    version="0.1.1",
```
Comment (Hwesta) [inline setup.py]: Since it has new features, 0.2 is more accurate.

**Comment by Hwesta** [comment]
:+1: We're starting to use metsrw more places, so it should have real versioning now.

**Pull request merged**